mithril-client/src/lib.rs

```diff
@@ -84,6 +85,7 @@ macro_rules! cfg_unstable {
 }
 
 pub mod aggregator_client;
+pub mod cardano_database_client;
```

Comment:
Is `#[cfg(feature = "unstable")]` needed?
```suggestion
#[cfg(feature = "unstable")]
pub mod cardano_database_client;
```